.prettierignore

```diff
@@ -1,5 +1,5 @@
-package.json
```

Comment:
Let's keep this here

Reply:
I tested it and `prettier` is treating `package.json` as a special json file, so the `files` key won't change to a single line